plugins/spark/spark-environment.xml.in

```diff
@@ -0,0 +1,1250 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Generated with envgen -->
```

Comment:
very minor, but this comment prob doesn't apply anymore, perhaps an updated comment would hep. <-- default environment for such a such build--> ?